doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -337,6 +337,14 @@ NFC samples
 
   * Added support for the ``nrf54l15dk/nrf54l15/cpuapp/ns`` board target.
 
+* :ref:`writable_ndef_msg` sample:
```

Comment:
You might also consider having both as the first entry like this:

* Added experimental ... to the following samples:

  * :ref:`writable_ndef_msg` sample
  * :ref:`nfc_shell` sample

Reply:
Sure, I will make a modification with next push